code/modules/griffening_cardgame/carddatums.dm

```diff
@@ -0,0 +1,685 @@
+	REMOVEDEF = 10
+	Attributes = list(GLOBAL_REMOVEDEF, REMOVEDEF_20_PER_TURN, FIRE, DISCARD_ON_AREA)
+
+/datum/griffeningcard/effect/authenticationdisk
+
+	name = "Authentication Disk"
+	desc = "This card can only be played if there's a captain in play. If the captain is on your side of the field, summon two security officer immediately on your side of the field either from your hand, deck or discard pile. If the captain is on the opponent's field, immediately summon two 'nuclear operatives' from your deck, hand or discard pile."
+	
+/datum/griffeningcard/effect/pinpointer
+
+	name = "Pinpointer"
+	desc = "When this card is played, put a 'authentication disk' card into your hand either from both yours and opponent's discard pile or deck."
+
+/datum/griffeningcard/effect/mattereater
+
+	name = "Matter Eater"
+	desc = "When this card is played, choose a face up equipment card and immediately send it to the gibbed pile and if you do, send this card to the discard pile."
+
+/datum/griffeningcard/equipment/spacesuit
+
+	name = "Space Suit"
+	desc = "Can only be equipped to humans, any human equipped with this is immune to 'hull breach', 'flamethrower', 'plasma fire' or 'incendiary grenade' as well as increase DEF by 10."
+
+/datum/griffeningcard/effect/dnaabsorbtion
+
+	name = "DNA Absorbtion"
+	desc = "Can only be played if you control a 'changeling', if you do, target a enemy creature and send it to the gibbed pile. If you do, send 'changeling' to the discard pile and immediately summon 'abomination' either from your hand, deck or discard pile."
+
+/datum/griffeningcard/effect/crematorium
+
+	name = "Crematorium"
+	desc = "When this card is played, place 5 cards from the discard pile of either player to their gibbed pile."
+
+/datum/griffeningcard/equipment/flamethrower
+
+	name = "Flamethrower"
+	desc = "When attacking a human using 'Flamethrower', reduce their DEF by 30 before attacking. If target humanoid is 'changeling', immediately destroy the 'changeling;. If the target is 'Abomination', reduce their DEF by half instead."
+
+/datum/griffeningcard/equipment/energyaxe
+
+	name = "Energy Axe"
+	desc = "Can only be equipped if you sacrifice a creature and if you do, you may equip this card to a creature. Raises the ATK by 40 and DEF by 20 of the equipped creature."
+
+/datum/griffeningcard/equipment/riotlauncher
+
+	name = "Riot Launcher"
+	desc = "If the equipped creature attacks an enemy that has an equipment card, remove the equipment card or choose one if there's more than one equipped."
+
+/datum/griffeningcard/equipment/telekinesis
+
+	name = "Telekinesis"
+	desc = "Requires a 'geneticist' to be in play on your field, take any equipment card currently equipped to a enemy creature and put it in your hand."
```

Comment:
Requires a geneticist in your control. Remove equipped equipment card from an enemy creature and place in your hand.